index.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+//Route to get a stock from the database. Calling https://codubee-projects-api.herokuapp.com/stocks/getStockData

```

Comment:
This comment may be a bit unnecessary here. Doesnt tell me much about the code that I cant figure out by reading the code.

Reply:
Agree with @Codubee